website/pages/en/querying/querying-best-practices.mdx

```diff
@@ -2,11 +2,9 @@
 title: Querying Best Practices
 ---
 
-The Graph provides a decentralized way to query data from blockchains.
+The Graph provides a decentralized way to query data from blockchains via GraphQL API, making it easier to query data with the GraphQL language.
```

Comment:
```suggestion
The Graph provides a decentralized way to query data from blockchains via GraphQL APIs, making it easier to query data with the GraphQL language.
```
Also, I find it interesting that we don't mention the word "subgraph" here... those GraphQL APIs are subgraphs :P

Reply:
Not sure if this was intentional (and if it was, don't know for what reason)